lib/web_ui/lib/src/engine/compositor/fonts.dart

```diff
@@ -40,10 +40,13 @@ class SkiaFontCollection {
     if (_unloadedFonts.isEmpty) {
       return;
     }
-
     final List<_RegisteredFont?> loadedFonts =
         await Future.wait(_unloadedFonts);
-    _registeredFonts.addAll(loadedFonts.where((x) => x != null));
+    for (_RegisteredFont? x in loadedFonts) {
```

Comment:
Rename `x` to something more descriptive?

Reply:
done.